examples/util.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package examples
+
+import (
+	"testing"
+
+	"github.com/pulumi/pulumi/pkg/v2/testing/integration"
+	"github.com/stretchr/testify/assert"
+)
+
+func dockerFileWithDependenciesOutputValidation(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+	expectedEphemeralDiagnosticsLines := []string{
+		"<{%reset%}>Building image '.'...<{%reset%}>\n",
```

Comment:
Do we need to keep these commented out lines?

Reply:
I left them there because that is the full output I currently get - but the test only validates a subset of the entries because others may not be stable in the future.  I think it's worth leaving the commented out lines to make it more clear what the lines that are being tests are.  I'll add a comment though.